atari-game-play/README.md

```diff
@@ -0,0 +1,46 @@
+# Atari - Solving Games with AI 🤖
+
+The purpose of this repository is to demonstrate how ZenML enables you to build complex reinforcement learning application in fairly simple way and we had made use of DQN Agent that solves Atari Games.

```

Comment:
Which game in particular? Is this Atari Breakout??

Reply:
Yes

Reply:
can you specify this in the README instead of talking about Atari Games in general?